_pages/emerging/clinical_decision_support_api.md

```diff
@@ -12,4 +12,6 @@ The guide covers triage of patients by non-clinical staff and triage of patients
 
 The Implementation Guide is based on initial discovery work, which informs the 1.0.0-alpha version of the Guide. The scope is expected to increase with more discovery work and as use cases are developed. Future versions of this Guide will include those updates as they are developed. Any resources not specifically mentioned in the Guide will follow the [HL7 FHIR STU3 guidance](https://www.hl7.org/fhir/stu3/index.html).
 
-[Access the Clinical Decision Support API implementation guide](https://developer.nhs.uk/apis/cds-api/)
\ No newline at end of file
+[Access version 1.0.0 the Clinical Decision Support API implementation guide](https://developer.nhs.uk/apis/cds-api-1-0-0/)
```

Comment:
**"of"** the Clinical Decision Support API....

Reply:
fixed